keys.txt

```diff
@@ -21,6 +21,7 @@
 "Toggle fullscreen" 1073741892
 "Toggle fast-forward" 1073741881
 "Fleet: Fight my target" 102
+"Fleet: Hold fire" 120
```

Comment:
I would suggest using the 'y' key for this. The other fleet commands are grouped reasonably close: D, F, G, and H, with toggle ammo usage being U.
```suggestion
"Fleet: Hold fire" 121
```